### PR TITLE
Add main page #9

### LINKED ISSUE #9: Add main page

### PR DESCRIPTION
**Pull request by sofronovsd**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
-    </div>
-  );
-}
+const App: React.FC = () => {
+    return (
+        <BrowserRouter>
+            <NavBar/>
+            <div className="container">
+                <Switch>
+                    <Route component={HomePage} path="/" exact/>
```
Comment (EldarZiyatdinov) [inline src/App.tsx:12]: what happens if you take a random path? for example "/randeom"

File: src/App.tsx [hunk 2]
```diff
@@ -1,11 +1,19 @@
 import React from 'react';
```
Comment (EldarZiyatdinov) [inline src/App.tsx:1]: please define before imports with an absolute path, then relative

File: src/components/home-page/HomePage.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+const HomePage: React.FC = () => {
+    return (
+        <div className="home-page">
+            <PostList posts={posts}/>
+        </div>
+    )
+};
+
+export default HomePage;
```
Comment (EldarZiyatdinov) [inline src/components/home-page/HomePage.tsx:13]: why do you use everywhere "export default"? what is the reason?

File: src/App.tsx [hunk 4]
```diff
@@ -1,11 +1,19 @@
...
+import {BrowserRouter, Switch, Route} from "react-router-dom";
 
-function App() {
-  return (
-    <div className="App">
-
-    </div>
-  );
-}
+const App: React.FC = () => {
```
Comment (EldarZiyatdinov) [inline src/App.tsx:6]: why do you define the function as constant?

File: src/sass/style.scss [hunk 5]
```diff
@@ -0,0 +1,2 @@
+@import "src/components/home-page/home-page";
```
Comment (EldarZiyatdinov) [inline src/sass/style.scss]: what is the purpose of this file?

File: src/components/nav-bar/NavBar.tsx [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import {NavLink} from "react-router-dom";
+
+const NavBar: React.FC = () => {
+    return (
+        <nav>
+            <div className="nav-wrapper">
+                <NavLink to="/" className="brand-logo center">Trip Viewer</NavLink>
+                <ul className="right">
+                    <li><NavLink to="/" exact>Home</NavLink></li>
```
Comment (EldarZiyatdinov) [inline src/components/nav-bar/NavBar.tsx:10]: why do you use NavLink instead of Link?

File: src/components/home-page/_home-page.scss [hunk 7]
```diff
@@ -0,0 +1,6 @@
+.home-page {
```
Comment (EldarZiyatdinov) [inline src/components/home-page/_home-page.scss:1]: why do you name the file with "_" at the beginning of the file name?

File: src/components/home-page/_home-page.scss [hunk 8]
```diff
@@ -0,0 +1,6 @@
...
+  display: flex;
```
Comment (EldarZiyatdinov) [inline src/components/home-page/_home-page.scss:2]: name file with ".module.scss" to use css modules

File: src/App.tsx (hunk 2 above)
Comment (sofronovsd) [reply src/App.tsx:1]: Could you please explain what do you mean? And for what?

File: src/components/home-page/HomePage.tsx (hunk 3 above)
Comment (sofronovsd) [reply src/components/home-page/HomePage.tsx:13]: Because I export only 1 object, so I will be able to simply import it

File: src/App.tsx (hunk 4 above)
Comment (sofronovsd) [reply src/App.tsx:6]: It's my code style

File: src/sass/style.scss (hunk 5 above)
Comment (sofronovsd) [reply src/sass/style.scss]: This file defines a style for the home page

File: src/components/nav-bar/NavBar.tsx (hunk 6 above)
Comment (sofronovsd) [reply src/components/nav-bar/NavBar.tsx:10]: With NavLink I can customize active class for active link.

File: src/components/home-page/_home-page.scss (hunk 7 above)
Comment (sofronovsd) [reply src/components/home-page/_home-page.scss:1]: As I know, when we are using "_" before filename, it tells a compiler that it should be proccessed  only where this file is imported.

File: src/components/home-page/_home-page.scss (hunk 8 above)
Comment (sofronovsd) [reply src/components/home-page/_home-page.scss:2]: I don't know anything about css modules. Why should I use them here?

File: src/App.tsx (hunk 1 above)
Comment (sofronovsd) [reply src/App.tsx:12]: I will add a redirect path.

**Pull request merged**